### ISSUE TITLE
FEWD / HW3

### ISSUE THREAD

**Issue by gafewd94** [Alihasadd/aVidler-fewd94#3]
Hey Alison,

Great start on hw #3! Looks like you still need to create the two column layout and add a number of CSS updates. If you need help with floats and clears or other aspects of laying out and styling your page, let me know and I'm happy to walk you through it.

Best,

Brian

**Comment by Alihasadd** [comment]
Yes! I was hoping to catch you after class last time but was really hungry! Maybe I can catch you before class today because I think that's an area where I'm having a few issues.

Thanks!!!
